js/common/lib/training-session-impl.ts

```diff
@@ -42,21 +49,140 @@ export class TrainingSession implements TrainingSessionInterface {
     }
   }
 
-  async loadParametersBuffer(_array: Uint8Array, _trainableOnly: boolean): Promise<void> {
-    throw new Error('Method not implemented.');
+  /**
+   * Helper function for runTrainStep and future runStep methods that handles the type-narrowing conversion from
+   * the given parameters to SessionHandler.FetchesType and RunOptions. Also handles type-checking to ensure that
+   * the inputs are correct.
+   *
+   * @param feeds the required input -- checked in this function for correctness.
```

Comment:
checked in this function for correctness. - what does this mean? Do you mean to say feeds validation happens in this function?

Reply:
Hmm, I removed the documentation about that -- I wanted to indicate that the method would throw an error if the inputs aren't one of the accepted types, but the validation that feeds contains the correct fields doesn't happen in this function.